pages/admin/directory.tsx

```diff
@@ -516,17 +479,87 @@ const MemberEdit: FC<{
     }
   };
 
-  const handleRegionChange = (value: string) => {
-    if (value === "New") {
-      console.log("create new region");
-      return;
+  const updateMemberField = async (uid: string, fieldName: string, newData: any, suggestedFilter?: boolean) => {
+    const response = await fetch("/api/update-member", {
```

Comment:
nit: prefer to use nouns when naming endpoints and use HTTP methods as the verbs. In this case, `PUT /api/member/{member_id}` will update the member. See my other comment about PUT vs. PATCH.

Reply:
Agreed, that seems to be a fairly standard way of naming the API though that may be a bit of a refactor for the rest of the APIs. 

If I don't add this change in this PR, I'll definitely file an issue to track this in a future PR

Reply:
Gonna put this one aside for the next PR to avoid growing the scope of this one

Tracking here: https://github.com/hawaiiansintech/hawaiiansintech/issues/32